csharp-intermediate-topics/ReadOnlyModifierInCSharp/ReadOnlyModifierInCSharp/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ReadOnlyModifierInCSharp
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
We can make this into a top level statement. This will simplify the file and reduce the nesting.